imap_processing/ultra/l1b/extendedspin.py

```diff
@@ -11,7 +11,8 @@
 from imap_processing.ultra.utils.ultra_l1_utils import create_dataset
 
 
-def calculate_extendedspin(
+def calculate_extendedspin(  # noqa: PLR0913
+    aux_dataset: xr.Dataset,
     hk_dataset: xr.Dataset,
     rates_dataset: xr.Dataset,
     de_dataset: xr.Dataset,
```

Comment:
If you wanted to get around the flag, you could pass in a dict of 
`input_datasets = {"aux": , "hk":, ...}`